Translations/WinMerge/French.po

```diff
@@ -242,7 +242,7 @@ msgid "&Vector Image Scaling"
 msgstr "Mise à l'échelle de l'image vectorielle"
 
 msgid "Compare Extracted &Text From Images"
-msgstr ""
+msgstr "Comparer le &texte extraits depuis d'images"
```

Comment:
Typos: Comparer le &texte extrait**s** depuis **d'**images -> Comparer le &texte extrait depuis les images

Reply:
Fixed in commit [a0b4ca6](https://github.com/WinMerge/winmerge/commit/a0b4ca64962010b5d2d50534026b5e14ebdd2d71)